module/profiler/profiler.go

```diff
@@ -135,18 +157,6 @@ func (p *AutoProfiler) pprofBlock(w io.Writer) error {
 	}
 }
 
-func (p *AutoProfiler) pprofMutex(w io.Writer) error {
```

Comment:
why was this removed?  if the gce uploader don't support this, maybe just don't upload it

Reply:
Will mention this in the description: "mutex" is almost a subset of "block" but less useful (it tracks where locks are held instead of where code is waiting (blocked) on acquiring the lock).